DrawingEngine/DrawingEngine.h

```diff
@@ -0,0 +1,25 @@
+#ifndef DRAWINGENGINE_H
+#define DRAWINGENGINE_H
+
+#include <iostream>
+#include <memory>
+#include <any>
+#include <vector>
+#include "../Logs/Logs.h"
+#include "../ClassFactory/ClassFactory.h"
+
+using namespace std;
+
+class DrawingEngine : public ClassFactory {
+private:
+	 shared_ptr<int> width = make_shared<int>();
+	 shared_ptr<int> height = make_shared<int>();
+
+public:
+	 DrawingEngine(any widthI, any heightI);
+
+	 virtual void print(vector<vector<int>> grid) = 0;
```

Comment:
DrawingEngine shall be able to draw the Drawables, not a buffer of ints.

The drawing engine shall have as a member a Framebuffer which would be a some type of container holding some representation of pixel, preferably (r,g,b) values